app/components/Input/index.js

```diff
@@ -29,27 +26,27 @@ function Input(props) {
 
   return (
     <Wrapper>
-      <SearchInputLabel>
-        <IconSpan
-          onKeyPress={() => {}}
-          role="button"
+      <SearchStyledLabel>
+        <IconButton
+          aria-label="search-button"
           onClick={() => setInputDisplay(!inputDisplay)}
-          tabIndex="0"
-        >
-          {searchIcon}
-        </IconSpan>
+          type="searchIcon"
+          colorSet="gray"
+          style={{ verticalAlign: 'middle' }}
```

Comment:
delete the inline style and add display: flex to the parent component to align the input and svgs